integration/commands_administration_test.go

```diff
@@ -773,6 +775,303 @@ func TestCommandsAdministrationDBStatsEmptyWithScale(t *testing.T) {
 	// https://github.com/FerretDB/FerretDB/issues/727
 }
 
+func TestCommandsAdministrationRenameCollection(t *testing.T) {
+	t.Parallel()
+
+	// rename collection must be executed from admin database.
+	adminDB := setup.SetupWithOpts(t, &setup.SetupOpts{DatabaseName: "admin"})
+
+	maxTableLen := strings.Repeat("a", 255)
```

Comment:
why `table` and not `collection`?

Reply:
My bad :see_no_evil: , wanted to make the CI not fail so I see codecov :see_no_evil: Yes max collection length is 235.